lib/src/Uncertainty/Algorithm/MetaModel/Kriging/GeneralLinearModelAlgorithm.cxx

```diff
@@ -562,8 +567,16 @@ Scalar GeneralLinearModelAlgorithm::maximizeReducedLogLikelihood()
   OptimizationProblem problem(reducedLogLikelihoodFunction);
   problem.setMinimization(false);
   problem.setBounds(optimizationBounds_);
-  solver_.setStartingPoint(initialParameters);
   solver_.setProblem(problem);
+  try
+  {
+    // If the solver is single start, we can use its setStartingPoint method
+    solver_.setStartingPoint(initialParameters);
+  }
+  catch (NotYetImplementedException & ex) // setStartingPoint is not implemented in MultiStart
```

Comment:
ex is unused, so you should write "catch (NotYetImplementedException &)", it introduces warnings in the windows build

Reply:
Hmm.. I simply copied https://github.com/openturns/openturns/blob/f2396a1ae442525f89d018e3bb3f627447c05647/lib/test/t_Sample_large.cxx#L54 
so I guess we should also change it in that file. Do you want me to do it in this PR?

Reply:
If you want yes

Reply:
OK: changed in all 4 files concerned + in t_Sample_large.cxx